ui/lib/core/addon/helpers/sync-destinations.ts

```diff
@@ -20,34 +19,49 @@ const SYNC_DESTINATIONS: Array<SyncDestination> = [
     icon: 'aws-color',
     category: 'cloud',
     maskedParams: ['accessKeyId', 'secretAccessKey'],
+    defaultValues: {
```

Comment:
Adding to the helper so that other default values dependent on type can be set here. For example, `secret_name_template` will potentially have configured defaults